docs/fsharp/tutorials/getting-started/getting-started-command-line.md

```diff
@@ -18,139 +18,133 @@ This article covers how you can get started with using F# on .NET Core 1.0 with
 
```

Comment:
The line above talks about Preview 2. The version should be removed the introduction and just be called out in the Prerequisites.